<<<<<<< HEAD
# Service to get the cartesian position of the robot

---
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
int64 ret
=======
# Service to get the cartesian position of the robot

---
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
int64 ret
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
string msg